encryptor/config/common/encryptedTypes.go

```diff
@@ -52,7 +53,7 @@ var (
 )
 
 // ValidateEncryptedType return true if value is supported EncryptedType
-func ValidateEncryptedType(value EncryptedType) error {
+func ValidateEncryptedType(value EncryptedType, tokenType common.TokenType) error {
```

Comment:
`TokenType` is not used. Or is it for the future?

Reply:
it's my fault. I tried to validate EncryptedType in pair with TokenType. But then found that we don't allow define token_type with data_type explicitly and this check is redundant. I removed it